src/Backend/Modules/ContentBlocks/Command/CreateContentBlock.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Command;
+
+use Backend\Core\Language\Locale;
+use Backend\Modules\ContentBlocks\Entity\ContentBlock;
+use Symfony\Component\Validator\Constraints as Assert;
+
+final class CreateContentBlock
+{
+    /**
+     * @var string
+     *
+     * @Assert\NotBlank()
+     */
+    public $title;
+
+    /**
+     * @var string
+     *
+     * @Assert\NotBlank()
+     */
+    public $template = ContentBlock::DEFAULT_TEMPLATE;
+
+    /**
+     * @var string
+     *
+     * @Assert\NotBlank()
+     */
+    public $text;
+
+    /**
+     * @var bool
+     */
+    public $isVisible = true;
+
+    /**
+     * @var ContentBlock
+     */
+    public $contentBlock;
+
+    /** @var Locale */
+    public $language;
+
+    /**
+     * @param Locale|null $language
+     */
+    public function __construct(Locale $language = null)
+    {
+        if ($language === null) {
+            $language = Locale::workingLocale();
```

Comment:
because the language now has a ValueObject called Locale so you can typehint etc on it
Locale::workingLocale() is a static constructor, it will create a new Locale instance for the current working language